diff --git a/node_modules/@imgly/editor-react-native/ios/IMGLYEditorModule.mm b/node_modules/@imgly/editor-react-native/ios/IMGLYEditorModule.mm
index 7c43b98..6fb526d 100644
--- a/node_modules/@imgly/editor-react-native/ios/IMGLYEditorModule.mm
+++ b/node_modules/@imgly/editor-react-native/ios/IMGLYEditorModule.mm
@@ -1,5 +1,5 @@
 #import "IMGLYEditorModule.h"
-#import "IMGLYEditorModule/IMGLYEditorModule-Swift.h"
+#import "IMGLYEditorModule-Swift.h"
 #import "IMGLYConstants.h"
 
 @implementation IMGLYEditorModule
